src/components/layout/GenericLayout/Navigation/index.tsx

```diff
@@ -1,17 +1,10 @@
-import React, { PropsWithChildren } from 'react'
 import styled from 'styled-components'
 
-import { MEDIA } from 'const'
-
-export const NavStyled = styled.ol`
+export const Navigation = styled.ol`
   list-style: none;
   display: flex;
   padding: 0;
 
-  /* @media ${MEDIA.mediumDown} {
-    margin: 0 0 0 auto;
-  } */
-
   > li {
     font-size: var(--font-size-larger);
```

Comment:
are you branched off theme with fonts? if so could you apply the font theming here? otherwise maybe leave a TODO

Reply:
Removed by instinct, will put it back so you can address it in your PRs